docs/deployment-examples/terraform-deployments/remote-state/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Terraform Remote State
+description: Using remote state with Octopus
+---
+
+When running Terraform on a local PC, the state of the resources managed by Terraform is saved in a local file. This state is queried to learn which resources already exist in order to properly apply updates and destroy resources.
```

Comment:
> the resources managed by Terraform is saved
I think it needs to be either "the resource is saved" or "the resources are saved"